modules/apps/product-navigation/product-navigation-personal-menu-api/src/main/java/com/liferay/product/navigation/personal/menu/PersonalMenuEntry.java

```diff
@@ -64,12 +66,17 @@ public String getPortletURL(HttpServletRequest request)
 	 * Returns <code>true</code> if the entry should be displayed in the
 	 * user personal menu.
 	 *
+	 *
+	 * @param  portletRequest the portlet request
 	 * @param  permissionChecker the permission checker
 	 * @return <code>true</code> if the entry should be displayed in the user
 	 *         personal menu; <code>false</code> otherwise
 	 * @review
 	 */
-	public default boolean isShow(PermissionChecker permissionChecker) {
+	public default boolean isShow(
+			PortletRequest portletRequest, PermissionChecker permissionChecker)
```

Comment:
Is there any reason we need to use `PortletRequest` here not `HttpServletRequest`?